bittrade_binance_websocket/messages/listen.py

```diff
@@ -3,4 +3,13 @@
 from reactivex import Observable, compose, operators, timer
 
 from elm_framework_helpers.websockets import models
-from elm_framework_helpers.websockets.operators import connection_operators
\ No newline at end of file
+from elm_framework_helpers.websockets.operators import connection_operators
+from elm_framework_helpers.websockets.models import WebsocketBundle, WEBSOCKET_MESSAGE, WEBSOCKET_HEARTBEAT, WEBSOCKET_STATUS, WEBSOCKET_OPENED, WEBSOCKET_CLOSED
+
+from bittrade_binance_websocket.models.enhanced_websocket import EnhancedWebsocket
+
+def filter_new_socket_only() -> Callable[[Observable[WebsocketBundle]], Observable[EnhancedWebsocket]]:
```

Comment:
I think this operator is already in elm_framework_helpers.websockets

Reply:
yes, i think there are quite a few needs to be there, but i have to put it here first to ensure all work well. will move it out later on